core/code/data_cache.js

```diff
@@ -24,13 +24,11 @@ window.DataCache = function() {
 
 }
 
-window.DataCache.prototype.store = function (qk, data, freshTime) {
+window.DataCache.prototype.store = function (qk, data) {
```

Comment:
Again, this is exposed function allowing everyone to set their data life expectancy.
Plugins might need and relly on that. We should be removing it even if it's not used internally.

Reply:
and again: don't developer for someone else or an unexpcted future 
(except there is a reasonable case)

it will just keep things complicated and un-maintainable